### PR TITLE
Update UserGuide to include "Update" section

### PR DESCRIPTION
**Pull request by Echomo-Xinyu**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -128,10 +129,61 @@ Examples:
...
 ```
 
 ---
+
+### Update student details: `update`
+
+Allows the user to update the students details including:
+
+- Phone number
+- Next of Kin’s number
```
Comment (gnahzeus) [inline docs/UserGuide.md:145]: Small detail - perhaps this field should be named Next of Kin's phone number for clarity?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -128,10 +129,61 @@ Examples:
...
+- Amount owed
+- Additional notes
+
+```yaml
+Notes:
+Student’s Name must be provided to uniquely identify the student.
+Student’s Name must exist in the records beforehand.
+```
+
+1. Phone number, Next of Kin’s number, Email, Address, and Class Date follow the same convention as in Adding a student:
```
Comment (gnahzeus) [inline docs/UserGuide.md:158]: Same with this line - perhaps this field should be named Next of Kin's phone number for clarity?

**Review by gnahzeus — COMMENTED** [review]
LGTM, except for one minute question!

File: docs/UserGuide.md (hunk 2 above)
Comment (Echomo-Xinyu) [reply docs/UserGuide.md]: Fix in later commit

File: docs/UserGuide.md (hunk 1 above)
Comment (Echomo-Xinyu) [reply docs/UserGuide.md]: Fixed in later commit

**Review by gnahzeus — APPROVED** [review]
LGTM!

**Pull request merged**